packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -173,27 +184,290 @@ class _PropertyInput extends StatelessWidget {
+            ),
+          ],
+        ),
+        Row(
+          children: [
+            Expanded(
+              child: _ExpandableWidgetDocumentation(
+                documentation:
+                    documentation ?? 'Creates ${addIndefiniteArticle(name)}.',
+              ),
+            ),
+          ],
+        ),
+        const PaddedDivider(),
+      ],
+    );
+  }
+}
+
+class _ExpandableWidgetDocumentation extends StatefulWidget {
+  const _ExpandableWidgetDocumentation({required this.documentation});
+
+  final String documentation;
+
+  @override
+  State<_ExpandableWidgetDocumentation> createState() =>
+      _ExpandableWidgetDocumentationState();
+}
+
+class _ExpandableWidgetDocumentationState
+    extends State<_ExpandableWidgetDocumentation>
+    with SingleTickerProviderStateMixin {
+  bool _isExpanded = false;
+  late AnimationController _expandAnimationController;
+  late Animation<double> _expandAnimation;
+
+  @override
+  void initState() {
+    super.initState();
+    _expandAnimationController = defaultAnimationController(this);
+    _expandAnimation = defaultCurvedAnimation(_expandAnimationController);
+  }
+
+  @override
+  void dispose() {
+    _expandAnimationController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final regularFontStyle = theme.regularTextStyle;
+    final fixedFontStyle = theme.fixedFontStyle.copyWith(
+      color: theme.colorScheme.primary,
+    );
+    final paragraphs = widget.documentation.split('\n');
+
+    if (paragraphs.length == 1) {
+      return RichText(
+        text: _convertDartDoc(
+          widget.documentation,
+          regularFontStyle: regularFontStyle,
+          fixedFontStyle: fixedFontStyle,
+        ),
+      );
+    }
+
+    final [firstParagraph, ...otherParagraphs] = paragraphs;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        // Currently this will process the Dart doc each time a user expands
+        // or collapses the text block. Because the Dart doc is never very
+        // large, this is not an expensive operation. However, we could
+        // consider caching the result if this needs to be optimized.
+        RichText(
+          text: _convertDartDoc(
+            firstParagraph,
+            regularFontStyle: regularFontStyle,
+            fixedFontStyle: fixedFontStyle,
+          ),
+        ),
+        if (_isExpanded)
+          FadeTransition(
+            opacity: _expandAnimation,
+            child: RichText(
+              text: _convertDartDoc(
+                otherParagraphs.join('\n'),
+                regularFontStyle: regularFontStyle,
+                fixedFontStyle: fixedFontStyle,
+              ),
+            ),
+          ),
+        GestureDetector(
+          onTap: _toggleExpansion,
+          child: Padding(
+            padding: const EdgeInsets.only(top: denseSpacing),
```

Comment:
nit: instead of including this padding widget, you could include a SizedBox as a child of the column. This would save a couple lines of code and an indentation.

Reply:
Ah good point, added a SizedBox